tests/unit/test_network_connections.py

```diff
@@ -69,56 +69,95 @@ def pprint(msg, obj):
 VALIDATE_ONE_MODE_NM = ARGS_CONNECTIONS.VALIDATE_ONE_MODE_NM
 
 ETHTOOL_FEATURES_DEFAULTS = {
+    "esp_hw_offload": None,
```

Comment:
Actually, the defaults should only contain the values with dashes (or only the values with underscores) but not both. So this change masks an implementation problem